content/agent/basic_agent_usage/windows.md

```diff
@@ -113,25 +130,7 @@ If you're running on a version older than 5.2 visit the status page in your web
 
 ## Configuration
 
-### For version >= 3.9.1
-
-You can use the Datadog Agent Manager located in the start menu to enable, disable and configure checks. You have to restart the Agent in order for your changes to be applied.
-
-### For version < 3.9.1
-
-The configuration file location depends on the version of Windows on which the Agent is installed. For Windows Server 2003, XP or older:
-
-  * Agent configuration:
-`C:\Documents and Settings\All Users\Application Data\Datadog\datadog.conf`
-  * Integration configuration:
-`C:\Documents and Settings\All Users\Application Data\Datadog\conf.d\`
-
-For Windows Server 2008, Vista and newer:
-
-  * Agent configuration:
-`C:\ProgramData\Datadog\datadog.conf`
-  * Integration configuration:
-`C:\ProgramData\Datadog\conf.d\`
+Use the Datadog Agent Manager located in the start menu to enable, disable and configure checks. You have to restart the Agent in order for your changes to be applied.
```

Comment:
serial comma
Use the Datadog Agent Manager located in the start menu to enable, disable, and configure checks. Restart the Agent in order for your changes to be applied.